genie-web/src/main/java/com/netflix/genie/web/dtos/ApiJobSubmission.java

```diff
@@ -0,0 +1,138 @@
+/*
+ *
+ *  Copyright 2019 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.web.dtos;
```

Comment:
those other packages, imo, should be renamed to the plural. This is `dtos` cause the rest of our packages, at least the ones that can be, are plural. So this is consistent with the web project at least.

I can't erally go back and change some of the ones in common right now as people using java client APIs rely on those packages so I would break their code.